nucypher/characters/lawful.py

```diff
@@ -869,18 +873,24 @@ def __init__(self,
             # Ursula is a Decentralized Worker
             #
             if not federated_only:
+
+                # Access staking node via node's transacting keys
+                transacting_power = TransactingPower(account=self.checksum_address,
+                                                     device=device,
+                                                     password=client_password,  # FIXME: password from somewhere
+                                                     blockchain=self.blockchain)
+                self._crypto_power.consume_power_up(transacting_power)
+
+                # Use blockchain power to substantiate stamp
+                self.substantiate_stamp(client_password=password)
```

Comment:
If the password was already provided when creating the `TransactingPower`, isn't here redundant? After all, stamp substantiation only makes sense when `TransactingPower` is active.